token/cli/src/config.rs

```diff
@@ -134,34 +134,45 @@ impl<'a> Config<'a> {
 
         let default_keypair = cli_config.keypair_path.clone();
 
-        let default_signer: Arc<dyn Signer> = {
+        let default_signer: Option<Arc<dyn Signer>> = {
             if let Some(owner_path) = matches.value_of("owner") {
                 signer_from_path_with_config(matches, owner_path, "owner", wallet_manager, &config)
+                    .ok()
             } else {
-                signer_from_path_with_config(
+                match signer_from_path_with_config(
                     matches,
                     &default_keypair,
                     "default",
                     wallet_manager,
                     &config,
-                )
+                ) {
+                    Ok(signer) => Some(signer),
+                    Err(e) => {
+                        if std::fs::metadata(&default_keypair).is_ok() {
+                            eprintln!("error: {}", e);
+                            exit(1);
+                        } else {
+                            None
+                        }
+                    }
+                }
```

Comment:
nit: To shorten this a bit, how about 
```suggestion
                .map_err(|e| if std::fs::metadata(&default_keypair).is_ok() {
                            eprintln!("error: {}", e);
                            exit(1);
                        } else {
                            e
                        }).ok();
```